src/hooks/useEditVideoSheet.tsx

```diff
@@ -20,6 +20,7 @@ export type EditVideoSheetTab = {
 }
 
 type EditVideoAssetsCache = Record<string, InputFilesState>
+type EditVideoTabCachedDirtyFormData = Record<string, Partial<EditVideoFormFields> & EditVideoSheetTab>
```

Comment:
Why do we need `EditVideoSheetTab` fields here? Aren't form fields enough?

Reply:
Yes I think after changing the `videoTabsCachedDirtyFormData` from an array to an object map EditVideoSheetTab should be enough